Powershell: Can't get the correct profile path

**Describe the bug**
Try to run `inshellisense bind` in powershell on macOS and got following error:
```
$ inshellisense bind
Select your desired shell for keybinding creation
  bash
  zsh
  fish
> powershell
  pwsh
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[Error: ENOENT: no such file or directory, open '/Users/username/Documents/WindowsPowershell/Microsoft.PowerShell_profile.ps1'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/Users/username/Documents/WindowsPowershell/Microsoft.PowerShell_profile.ps1'
}
```
**To Reproduce**
Steps to reproduce the behavior:
1. Open powershell  on macOS
2. Run `inshellisense bind`

**Expected behavior**
The correct profile path is `~/.config/powershell/Microsoft.PowerShell_profile.ps1`
```
$ file $PROFILE                                                  
/Users/user/.config/powershell/Microsoft.PowerShell_profile.ps1: ASCII text
```

**Environment**
 - OS: macOS
 - Output of `inshellisense --version`: 0.0.1-rc.1
 - Nodejs Version: 21.1.0

**Additional context**
Add any other context about the problem here.

After manually insert the key-bindings into the profile,:
```powershell
if(Test-Path '/Users/username/.inshellisense/key-bindings-pwsh.ps1' -PathType Leaf){. /Users/username/.inshellisense/key-bindings-pwsh.ps1}
```
I got another error:
```
node:internal/modules/cjs/loader:1147
  throw err;
  ^

Error: Cannot find module '/Users/username/\AppData\Roaming\npm\node_modules\@microsoft\inshellisense\build\index.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```
It seems that there are some hard-code paths for windows...

--- a/src/utils/bindings.ts
+++ b/src/utils/bindings.ts
@@ -20,7 +20,9 @@ export enum Shell {
   Fish = "fish",
 }
 
-export const supportedShells = [Shell.Bash, Shell.Powershell, Shell.Pwsh, Shell.Zsh, Shell.Fish];
+export const supportedShells = [Shell.Bash, process.platform == "win32" ? Shell.Powershell : null, Shell.Pwsh, Shell.Zsh, Shell.Fish].filter(
+  (shell) => shell != null,
+) as Shell[];
 
 const bashScriptCommand = (): string => {
   return `[ -f ~/${cacheFolder}/key-bindings.bash ] && source ~/${cacheFolder}/key-bindings.bash`;
@@ -93,13 +95,15 @@ export const availableBindings = async (): Promise<Shell[]> => {
     }
   }
 
-  const powershellConfigPath = path.join(os.homedir(), "Documents", "WindowsPowershell", "Microsoft.PowerShell_profile.ps1");
-  if (!fs.existsSync(powershellConfigPath)) {
-    bindings.push(Shell.Powershell);
-  } else {
-    const powershellConfigContent = fsAsync.readFile(powershellConfigPath, { encoding: "utf-8" });
-    if (!(await powershellConfigContent).includes(powershellScriptCommand())) {
+  if (process.platform == "win32") {
+    const powershellConfigPath = path.join(os.homedir(), "Documents", "WindowsPowershell", "Microsoft.PowerShell_profile.ps1");
+    if (!fs.existsSync(powershellConfigPath)) {
       bindings.push(Shell.Powershell);
+    } else {
+      const powershellConfigContent = fsAsync.readFile(powershellConfigPath, { encoding: "utf-8" });
+      if (!(await powershellConfigContent).includes(powershellScriptCommand())) {
+        bindings.push(Shell.Powershell);
+      }
     }
   }
 

